behavior/MouseTracking/get_mouse_XY_pos.m

```diff
@@ -28,10 +37,11 @@
         if strcmp(option,'displayTracking')
             display_tracking = 1;
             figure;
-            fprintf('Displaying live tracking...');
+            fprintf('Displaying live tracking...');            
         else
-            fprintf('Did not recognize %s.',option);
-            return
+            trial_aware = 1;
+            trial_indices = get_trial_frame_indices(option);
+            trial_indices = trial_indices(:,[1 4]);
```

Comment:
FYI, the downside of structuring `varargin` processing this way is that you can't do "live tracking" if you are providing the trial indices text file.